mass_mailing_count/l18n/sv_SE.po

```diff
@@ -0,0 +1,34 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* mass_mailing_count
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-12-07 10:15+0000\n"
+"PO-Revision-Date: 2021-12-07 10:15+0000\n"
+"Last-Translator: <Haze Lidman haze.lidman@arbetsformedlingen.se>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: mass_mailing_count
+#: code:addons/mass_mailing_count/models/mass_mailing.py:23
+#, python-format
+msgid "Config parameter mass_mailing_allowed_search_groups is not set contact your administrator"
+msgstr ""
+
+#. module: mass_mailing_count
+#: model:ir.model,name:mass_mailing_count.model_mail_mass_mailing
+msgid "Mass Mailing"
+msgstr "Massutskick per e-post"
+
+#. module: mass_mailing_count
+#: code:addons/mass_mailing_count/models/mass_mailing.py:28
+#, python-format
+msgid "You have to be member of the group(s) {groups}."
+msgstr "Du måste vara medlem i grupperna "
```

Comment:
{groups} is missing, which means that the actuall groups that are needed is not displayed.